scripts/vast-du

```diff
@@ -19,5 +19,13 @@ else
 	DU="du"
 fi
 
+# Run vast and filter for rate and latency results
 vast "${@}" 2>&1 >/dev/null | grep "rate\|latency"
+# Query vast for average node throughput rate
+if [[ "$*" == *import* ]]; then
+	NODE_THROUGHPUT="$(vast -N export json '#type == "vast.account" && key == "node_throughput.rate"' 2>/dev/null \
+		| jq -s 'map(.value | tonumber) as $rates | $rates | add / ($rates | length)')"
```

Comment:
This computes the mean, which I don't think is robust statistic because it assumes normality. Can report min, max, mean, and median?

Reply:
It's an estimate currently anyways, because `node_throughput.duration` is not necessarily exactly 1 second. 

Do you think reporting min, max, mean, and median exactly makes sense here?

Reply:
I'm not sure if I understand your comment. Why does it matter whether the data is a set of estimates? My comment was more about what we are doing with the set.

If we have a uniform or normal distribution, the mean is surely not wrong, but I don't know the shape of the data and therefore would like to report defensively at first.

Reply:
Should've clarified better. The data is already kind of skewed because not every rate was measured over the same duration, it's 1.05s ± 0.05s in practice.

Obviously I can write a small jq script to calculate the data you request right now. Will do in a bit..

Reply:
This has been addressed, but the median is still obviously skewed because the rates are not normalized with their actual duration.